Configs/.config/hypr/scripts/screenshot.sh

```diff
@@ -7,20 +7,27 @@ fi
 ScrDir=`dirname $(realpath $0)`
 source $ScrDir/globalcontrol.sh
 swpy_dir="$HOME/.config/swappy"
-save_dir="${2:-$XDG_PICTURES_DIR}"
+save_dir="${2:-$XDG_PICTURES_DIR/Screenshots}"
 save_file=$(date +'%y%m%d_%Hh%Mm%Ss_screenshot.png')
 mkdir -p $save_dir
 
+temp_screenshot="/tmp/screenshot.png"
 case $1 in
-p)  grim $save_dir/$save_file ;;
+# print all outputs
+p)  grimblast copysave screen $temp_screenshot && swappy -f $temp_screenshot ;;
+# drag to manually snip an area / click on a window to print it
 s)  mkdir -p $swpy_dir
     echo -e "[Default]\nsave_dir=$save_dir\nsave_filename_format=$save_file" > $swpy_dir/config
```

Comment:
this line should be before the case statement for swappy to work, else it will overwrite previous file.